overlord/hookstate/ctlcmd/services_test.go

```diff
@@ -447,7 +447,7 @@ func (s *servicectlSuite) TestQueuedCommandsSingleLane(c *C) {
 	s.st.Lock()
 
 	chg := s.st.NewChange("install change", "install change")
-	ts, err := snapstate.Install(s.st, "one", &snapstate.RevisionOptions{Revision: snap.R(1)}, 0, snapstate.Flags{})
+	ts, err := snapstate.Install(context.TODO(), s.st, "one", &snapstate.RevisionOptions{Revision: snap.R(1)}, 0, snapstate.Flags{})
```

Comment:
this isn't really a TODO though :-) or is it?

Reply:
Not really, I was following the exiting UpdateMany() whichhas a TODO context but that is wrong as well so fixing them all now :)